tests/test_dockerfile.py

```diff
@@ -0,0 +1,392 @@
+import sys
+import os
+import pytest
+from entrypoint_hook import CommandNotFound
+
+EXECUTABLES_FOLDER = "/usr/local/bin/"
+
+def abs_path(executable):
+    """Build manually (expected) executable absolute path"""
+    return os.path.join(EXECUTABLES_FOLDER, executable)
```

Comment:
This would add a `\` when ran on windows, I think that's not what we're looking for here?

Reply:
Depending on what we want to do, maybe. The answer may differ if we can run those tests from outside a container.